trade-finance-manager-api/src/v1/controllers/tfm.controller.js

```diff
@@ -1,6 +1,7 @@
 const activity = require('../helpers/activity');
 const api = require('../api');
 
+// This is never used - should I delete it?
```

Comment:
Ah yes the function from deal.controller.js is the only one used right? Happy for this to be deleted